make.go

```diff
@@ -21,6 +22,24 @@ func commandOutput(name string, args ...string) string {
 	return strings.TrimSpace(string(out))
 }
 
+// List all the go files in the current directory, excluding this file
+func listFilesToBuild() []string {
```

Comment:
Actually after digging a bit it looks like there's no need to list files to the `go build` command, the default behavior works fine. I'll remove al this logic.